app/Http/Controllers/EmailController.php

```diff
@@ -27,6 +27,37 @@ public function index()
         ]);
     }
 
+    /** @return View */
+    public function filter(Request $request)
+    {
+        $filteredEmails = Email::orderBy('id', 'desc');
+        $description = $request->has('search_description');
+        $subject = $request->has('search_subject');
+        $body = $request->has('search_body');
+        $searchTerm = $request->input('searchterm');
+
+        if ($description) {
+            $filteredEmails = $filteredEmails->where('description', 'LIKE', '%'.$searchTerm.'%');
+        }
+
+        if ($subject) {
+            $filteredEmails = $filteredEmails->where('subject', 'LIKE', '%'.$searchTerm.'%');
+        }
+
+        if ($body) {
+            $filteredEmails = $filteredEmails->where('body', 'LIKE', '%'.$searchTerm.'%');
+        }
```

Comment:
Don't you overwrite the filteredEmails here so you practically can only use 1 filter at a time?

Reply:
Yes, you are right, meant to use an orWhere()